examples/increment-only-counter/README.md

```diff
@@ -0,0 +1,36 @@
+# Increment-only counter
+
+This example implements an in-memory key-value cache.
+Keys are strings, and values are increment-only counters.
```

Comment:
FWIW, I think we should be implementing a delta-enabled increment-only counter as the basic example. Or, if you really must, a grow-only map of these.